src/components/dashboardTemp.tsx

```diff
@@ -0,0 +1,73 @@
+import { Box, Card, Flex, Heading, Text } from "@chakra-ui/react";
+import type { DashboardData } from "types/bribersDashboard.trpc";
+
+interface Props {
+  data: DashboardData;
+  requestedRound: number;
+}
+
+export const DashboardTemp: React.FC<Props> = ({ data, requestedRound }) => {
```

Comment:
just a temporary copy of previous code, will be removed, if everything new works